### ISSUE TITLE
FLoC calculation on publishers' websites only

### ISSUE THREAD

**Issue by Myrtle** [WICG/floc#82]
Hi!

(Greg Truchetet From Teads here)

We were reading this [article](https://web.dev/floc/) yesterday and something is specified that was not fully obvious before.
It is said that:
```
For the FLoC origin trial in Chrome 89, websites that don't opt out will be included in the FLoC calculation if Chrome detects that they are sites which load ads.
```

So it means only the publishers' websites are going to be taken into account if they didn't opt-out. The advertisers' websites are going to be ignored.
Can you give more information on the rationale behind this decision?
One of the reasons we have in mind would be to preserve the "interest-based" nature of the cohorts and to not "pollute" them with intents, letting `FLEDGE` exclusively covering this topic.
It is also stated "for the trial", do you plan to try to include the advertisers' traffic in the computation at some point or this information was not important more than that? :)

Thank you!

**Comment by jkarlin** [comment]
Hmm, that article should probably read "ads or ad script". So advertisement sites will indeed often be included as they load ad script to report statistics such as conversions. The rationale is that we wanted to restrict pages included in the FLoC calculation in the OT to roughly where we'd expect to see them in the future once it's fully launched and ad-tech is using it, namely sites that are interested in advertising.

**Comment by michaelkleber** [comment]
Good point, thank you @Myrtle.  Hey @samdutton, can we get that web.dev article changed to say "will be included in the FLoC calculation if Chrome detects that they are sites which load ads _or ads-related resources_"?  I'm sure other people will have the same question.

**Comment by Myrtle** [comment]
Thank you for your quick replies @jkarlin and @michaelkleber, adding the `ads-related resources` precision sounds a good plan indeed. :)

**Comment by samdutton** [comment]
Thanks @Myrtle – meant to reply that I fixed this last week (in [this PR](https://github.com/GoogleChrome/web.dev/pull/5007/files)).

**Comment by johansmith90** [comment]
I was wondering if you could clarify a little bit further as to the precise definition of `ads-related resources`? For example, is Google Analytics Javascript considered to be an ads-related resource? What about embedded Youtube iframes? I read the Chromium documentation about [Ad Tagging](https://chromium.googlesource.com/chromium/src/+/master/docs/ad_tagging.md), but was not able to find an exhaustive list of `ads-related resources` therein. Thank you kindly in advance for your time and help.

**Comment by michaelkleber** [comment]
Hi @johansmith90: Check out the answer from our Dev Rel folks on Twitter: https://twitter.com/rowan_m/status/1383380295336878091

The precise list we use starts from EasyList but then goes through some crunching before ending up in Chromium, so using the browser dev tools to see exactly what's ads-related on any particular page is probably the easiest way to answer all your questions.

**Comment by jen801** [comment]
Duplicate of #

**Comment by Mbellsudteen** [comment]
yutthayongranee@gmail.com

**Comment by Mbellsudteen** [comment]
#134

**Comment by MH621003** [comment]
Good

**Comment by 8594085786** [comment]
https://github.com/WICG/floc/issues/82#issuecomment-811105792

**Comment by MH621003** [comment]
Hello, 👋. 
I wish you both well and bless you for the year..
